UnityProject/Assets/Scripts/Chat/DiscordWebhook.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections.Specialized;
+using System.Net;
+using System.Text.RegularExpressions;
+
+
+namespace DiscordWebhook
+{
+	public static class DiscordWebhookMessage
+	{
+		public static byte[] Post(string url, NameValueCollection pairs)
+		{
+			using (WebClient webClient = new WebClient())
+				return webClient.UploadValues(url, pairs);
```

Comment:
The return from this isn't used anywhere as far as I can see.

Reply:
removed